build.gradle

```diff
@@ -206,5 +186,5 @@ signing {
 
 clean {
     delete genDir
-    delete sharedDir
+    delete sharedDir 
```

Comment:
I think we should not clear the shared dir because it contains dependencies we might still need